tests/acceptance/pageObjects/favoritesPage.js

```diff
@@ -13,6 +13,20 @@ module.exports = {
       return this.navigate(this.url()).waitForElementPresent(
         this.page.FilesPageElement.filesList().elements.anyAfterLoading
       )
+    },
+    getBatchActionButtonPresentStatus: async function () {
+      let presentStatus = null
+      await this.isVisible('@batchActionButton', 'suppressNotFoundErrors', (res) => {
+        presentStatus = res.status
+      })
+      return presentStatus
+    }
+  },
```

Comment:
```suggestion
    getBatchActionButtonVisibilityStatus: async function () {
      let presentStatus = null
      await this.isVisible('@batchActionButton', 'suppressNotFoundErrors', (res) => {
        presentStatus = res.status
      })
      return presentStatus
    }
  },
```
since you're cheking if the element is visible or not ;)